dist/FormView.js

```diff
@@ -42,7 +42,7 @@
 
       //Attach Events to preexisting elements if we don't have a template
       if (!this.template) this.runInitializers();
-      this.on('item:rendered',this.runInitializers, this);
```

Comment:
@iamchrismiller and @jgruhl I changed a version into [`bower.json`](https://github.com/viverae/marionette.formview/pull/12/files#diff-0a08a7565aba4405282251491979bb6bR3)